samples/MSALAngularDemoApp/src/app/app.component.ts

```diff
@@ -3,6 +3,7 @@ import {BroadcastService} from "@azure/msal-angular";
 import { MsalService} from "@azure/msal-angular";
 import {ProductService} from "./product/product.service";
 import {Subscription} from "rxjs/Subscription";
+import { AuthError, AuthResponse } from '../../../../lib/msal-core/lib-commonjs';
```

Comment:
can this work with lerna and not reaching in?

Reply:
Ah this was VS Code auto-import shenanigans, I'll fix.